modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/custom_meta_tags.jsp

```diff
@@ -80,10 +80,10 @@ if (Validator.isNull(backURL)) {
 				type="submit"
 			/>
 
-			<clay:link
+			<clay:button
 				displayType="secondary"
-				href="<%= HtmlUtil.escape(backURL) %>"
 				label='<%= LanguageUtil.get(request, "cancel") %>'
+				onclick='<%= "location.href=\'" + HtmlUtil.escape(backURL) + "\'" %>'
```

Comment:
We should use `Liferay.Util.navigate()` here, instead of `location.href`. This will result in better behavior with SPA turned on.

The same in two other places in PR.

Reply:
Thank you!